.gitignore

```diff
@@ -9,3 +9,4 @@ org.eclipse.jdt.apt.core.prefs
 org.eclipse.m2e.core.prefs
 org.eclipse.pde.core.prefs
 
+wiquery-*.i*
```

Comment:
I use IntelliJ for development.
And it creates some files with .ipr, .iml and .iws ending.
Just didn't want to push them.
You can revert this particular change if you want to :)